modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -571,6 +571,18 @@
     end
   end
 
+  context 'with arrived status' do
+    let!(:arrived_appt) do
```

Comment:
Is there a reason for the bang here?

Reply:
bang made it easier to debug but not necessary so removed it from the other spec I got it from too.